community/libtbb/build

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+patch -p1 < musl.patch
+
+sed -i "s|CPLUS = g..|CPLUS = $(CXX)|g"                         build/linux.gcc.inc
+sed -i "s|CONLY = gcc|CONLY = $(CC)|g"                          build/linux.gcc.inc
+sed -i "s@#define MALLOC_UNIXLIKE_OVERLOAD_ENABLED __linux__@@" src/tbbmalloc/proxy.h
+
+make
+
+mkdir -p "$1/usr/lib"
+mv build/linux_*/*.so* "$1/usr/lib"
+mkdir -p "$1/usr/include"
+mv include/tbb "$1/usr/include"
```

Comment:
As per the KISS Style Guide, this could be better organised as;
```shell
mkdir -p "$1/usr/lib"
mkdir -p "$1/usr/include"

mv build/linux_*/*.so* "$1/usr/lib"
mv include/tbb "$1/usr/include"
```

make sure that there is a space between the commands `mkdir` and `mv`